whitepapers/game-theory.md

```diff
@@ -0,0 +1,121 @@
+## The Prisoner's Dilemma - A Simultaneous Game
+
+The first game any student of game theory learns is the prisoner's dilemma.  This is due to its simplicity and it's applications to a wide variety of strategic situations. Once you see and understand this game, you will see it at play everywhere.   
+### Context  
+The story goes like this; Two thieves plan to rob a store. As they approach the door, the police arrest them for trespassing. The police suspect that the pair planned to rob the store but they lack the evidence to prove it. They therefore require a confession to charge the suspects with the more serious crime.
+The interrogator separates the suspects and tells them each:
+
+*“We are charging you with trespassing which will land you a one month jail sentence. I know you were planning to rob the store but I can’t prove it* *without your testimony. Confess to me now, and I will dismiss your trespassing charge and set you free. Your friend will be charged for attempted* *robbery and face 12 months in jail.* 
+*I’m offering your friend the same deal. If you both confess, your testimony is no longer as valuable and you will both receive 8 months in jail.”*
+
+Both players are self-interested and want to minimise their jail time. What should they do? 
+
+### Payoff Matrix
+
+Using a payoff matrix allows us to condense all the information into an easy-to-analyse diagram:  
+![](../.gitbook/assets/matrix_1.png)
+
+Player 1’s available strategies are the rows (Quiet or Confess) and their corresponding payoffs are the first numbers in each cell. 
+Players 2’s available strategies are the columns and their corresponding payoffs are the second numbers in the cells. 
+If player 1 stays Quiet and player 2 stays Quiet the game ends in the top left corner of the matrix. If both players Confess the game ends in the bottom right corner of the matrix and so on. 
+
+### Strategies
+To see which strategy each player will choose we should look at each move in isolation. From player 1’s perspective, what should he do if he thinks player 2 will stay Quiet?    
+![](../.gitbook/assets/matrix_2.png)
+
+We can see that player 1 should Confess because if he stays Quiet he will get one month in jail and we have already stated that both players prefer less time in jail. 
+What about if player 1 thought that player 2 was going to Confess?    
+![](../.gitbook/assets/matrix_3.png)
+
+Again it seems that player 1 should Confess as it leads to 8 months jail time rather than the 12 on offer if player 1 stays Quiet. 
+Putting this together we reach an important conclusion:  
+**Player 1 is better off confessing regardless of player 2’s strategy.**  
+
+Let’s look at player 2’s perspective, assuming he thinks player 1 will stay quiet:    
+![](../.gitbook/assets/matrix_4.png)
+
+Looking at the second numbers now we can see that like player 1, player 2 should confess as well: he will be set free instead of getting 1 month in jail.     
+![](../.gitbook/assets/matrix_5.png)
```

Comment:
```suggestion
Let’s look at player 2’s perspective, assuming he thinks player 1 will stay quiet:

![](../.gitbook/assets/matrix_4.png)

Looking at the second numbers now we can see that like player 1, player 2 should confess as well: he will be set free instead of getting 1 month in jail.

![](../.gitbook/assets/matrix_5.png)
```